tests/unit/algorithm/hull/ConcaveHullTest.cpp

```diff
@@ -243,6 +243,28 @@ void object::test<15>()
        0, "POLYGON ((20 90, 40 96, 56 95, 70 80, 80 90, 90 70, 80 60, 95 45, 80 40, 70 20, 90 20, 80 10, 60 15, 45 5, 40 20, 40 80, 15 45, 21 30, 20 10, 10 20, 5 40, 11 60, 20 70, 20 90))" );
 }
 
+//------------------------------------------------
+
+// These tests test that the computed Delaunay triangulation is correct
+// See https://github.com/locationtech/jts/pull/1004
+  
+// testRobust_GEOS946
+template<>
+template<>
+void object::test<20>()
```

Comment:
should these numbers be reset to 16? Or leave as-is in-case there are other backports from main (which has the missing 16-19 tests). It doesn't seem to mind that there are gaps of test numbers.

Reply:
I've decided to leave these alone. Previously I would have reset them, but it makes potential other backports difficult to place in if they have tests before/after these.